src/HucaresServer.UnitTests/DetectedPlateControllerTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿using FakeItEasy;
+using HucaresServer.Controllers;
+using HucaresServer.Storage.Helpers;
+using HucaresServer.Storage.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using static HucaresServer.Models.CameraInfoDataModels;
+
+namespace HucaresServer.Storage.UnitTests
+{
+    [TestClass]
+    public class DetectedPlateControllerTests
+    {
+
+        [TestMethod]
+        public async Task GetAllDetectedMissingPlates_WhenCalled_ShouldCallHelper()
+        {
+            //Arrange
+            var fakeDetectedPlateHelper = A.Fake<IDetectedPlateHelper>();
+
+            var expectedDLPList = new List<DetectedLicensePlate>() { new DetectedLicensePlate() { Id = 0 } };
+            A.CallTo(() => fakeDetectedPlateHelper.GetAllDetectedMissingPlates())
+                .Returns(expectedDLPList);
+
+            var cameraController = new DetectedPlateController() { DetectedPlateHelper = fakeDetectedPlateHelper, Request = new HttpRequestMessage() };
```

Comment:
Why is the name cameraController, if the object created is DetectedPlateController?